### DISCUSSION TITLE
Github Codespaces graphics

### DISCUSSION THREAD

**Discussion by jujumumu** [community/community#46194] [Codespaces] [answered]
### Select Topic Area

Question

### Body

I tried an application that uses graphics, results in ERROR: X11 Display is not available
Is there any way I can get graphics working in GitHub codespaces? What if I applied for a GPU machine also, would that allow for graphics?

**Comment by jge162** [comment, marked as answer]
GitHub Codespaces does not currently support graphics-intensive applications that require a GUI, such as applications that require an X11 display. Codespaces is designed to be a cloud-based development environment that does not support GUI applications.

**Reply by jujumumu** [reply]
Does github ever plan on making it support this type of work in codespaces?

**Reply by jge162** [reply]
I do not know
